packages/components/src/components/send-email-template/send-email-template.scss

```
@@ -3,15 +3,13 @@
     padding: 0 16px;
 
     &__title {
```

You can combine both title and icon into a single line selector.
e.g - 
```suggestion
    &__title, &__icon {
        margin-bottom: 2.4rem;
    }
```